src/views/Wallet/views/Dashboard/index.js

```diff
@@ -26,13 +28,32 @@ const StyledP = styled(Paragraph)`
     text-align: center;
 `;
 
+const Overlay = styled.div`
+    display: flex;
+    width: 100%;
+    height: 40px;
+    justify-content: center;
+    align-items: center;
+    opacity: 0.2;
+    background: white;
+`;
+
+const Image = styled.img``;
+
+const EthImage = styled(Image)`
+    margin-right: 10px;
```

Comment:
Isn't it better to add margin-right to every coin image except the last one?

Reply:
sure :) done